index.js

```diff
@@ -7,11 +7,17 @@
 const fs = require('fs')
 const path = require('path')
 const args = process.argv.slice(2)
+const root = process.cwd()
 
 args.forEach(function (map) {
-  const root = process.cwd()
   const data = map.split(':')
-  const source = path.resolve(root, data[0])
+
+  if (data.length !== 2) {
+    throw new Error('Argument must be "<string>:<string>"')
+  }
+
+  const absSource = path.resolve(root, data[0])
+  const relSource = path.relative(root, absSource)
```

Comment:
Of course I could've just used `data[0]` as-is.

One the one hand, this code would shorten needlessly long paths. On the other hand, it removes flexibility from the user.